spec.html

```diff
@@ -12770,33 +12770,33 @@
         </ul>
       </emu-clause>
       <emu-clause id="sec-applying-the-exp-operator">
-        <h1>Applying the `**` operator</h1>
+        <h1>Applying the `**` Operator</h1>
```

Comment:
The spec isn’t consistent as to whether title case or sentence case should be used for headings. E.g. https://tc39.github.io/ecma262/#sec-normative-references uses sentence case, but https://tc39.github.io/ecma262/#sec-ecmascript-overview uses title case. (I’d personally prefer sentence case everywhere.)